controller/src/main/java/com/softserve/borda/config/jwt/JwtConvertor.java

```diff
@@ -2,16 +2,19 @@
 
 import com.softserve.borda.entities.JWTUser;
 import com.softserve.borda.entities.User;
+import com.softserve.borda.exceptions.CustomEntityNotFoundException;
 import com.softserve.borda.repositories.JWTUserRepository;
 import org.springframework.stereotype.Service;
 
 @Service
 public class JwtConvertor {
 
-    private final com.softserve.borda.config.jwt.JwtProvider jwtProvider;
+    private final JwtProvider jwtProvider;
 
     private final JWTUserRepository jwtUserRepository;
 
+    private static final int JWT_START = 7;
```

Comment:
Does api use it somewhere else?
If not then remove that property.

Reply:
It uses it in the same file:
return jwtUserRepository.findById(jwt.substring(JWT_START)).orElseThrow(
                () -> new CustomEntityNotFoundException(User.class)).getUser();

We pass jwt token from header, which always has "Bearer {token}", so we cut off the starting part, leaving only token. Property was created to avoid usage of 'Magic numbers'

Reply:
Fixed